ccm_web/client/src/components/BulkSectionCreateFileExample.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import { FILE_HEADER_VALUE } from '../pages/BulkSectionCreate'
```

Comment:
I see an error complaining on this line say this

```
ERROR in /base/client/src/components/BulkSectionCreateFileExample.tsx
ccm_web     | ./client/src/components/BulkSectionCreateFileExample.tsx 2:9-26
ccm_web     | [tsl] ERROR in /base/client/src/components/BulkSectionCreateFileExample.tsx(2,10)
ccm_web     |       TS2614: Module '"../pages/BulkSectionCreate"' has no exported member 'FILE_HEADER_VALUE'. Did you mean to use 'import FILE_HEADER_VALUE from "../pages/BulkSectionCreate"' instead?
```

Reply:
Sorry that's fixed